lib/urlwatch/filters.py

```diff
@@ -406,6 +408,62 @@ def _to_string(self, element):
 
         return etree.tostring(element, pretty_print=True, method=self.method, encoding='unicode', with_tail=False)
 
+    @staticmethod
+    def _remove_element(element):
+        parent = element.getparent()
+        if parent is None:
+            # Do not exclude root element
+            return
+        if isinstance(element, etree._ElementUnicodeResult):
+            if element.is_tail:
+                parent.tail = None
+            elif element.is_text:
+                parent.text = None
+            elif element.is_attribute:
+                del parent.attrib[element.attrname]
+        else:
+            previous = element.getprevious()
+            if element.tail is not None:
+                if previous is not None:
+                    previous.tail = previous.tail + element.tail if previous.tail else element.tail
+                else:
+                    parent.text = parent.text + element.tail if parent.text else element.tail
+            parent.remove(element)
+
+    @staticmethod
+    def _reevaluate(element):
```

Comment:
Since this function uses `LxmlParser._orphaned`, this should be a `@classmethod`:

```
@classmethod
def _reevaluate(cls, element):
    if cls._orphaned(element):
        return None
    ...
```